### DISCUSSION TITLE
Preflight

### DISCUSSION THREAD

**Discussion by kmarler18** [tailwindlabs/tailwindcss#10051] [Help] [answered]
Hi All,

New to Tailwind, thanks for your time.

Is what I'm looking at in the screenshot below '[Preflight](https://tailwindcss.com/docs/preflight#overview)' that's been automatically added via the '[Play CDN Script](https://tailwindcss.com/docs/installation/play-cdn)'?

![Screen Shot 2022-12-10 at 3 29 01 PM](https://user-images.githubusercontent.com/96091818/206874194-2f64c1cd-3342-4f42-a031-a93def1f7d58.png)

Thank you so much,
Kyle

**Comment by wongjn** [comment, marked as answer]
That `<style>` block is all the CSS generated by Tailwind which would include the Preflight styles if you have not specifically disabled the Preflight core plugin.

**Reply by kmarler18** [reply]
Quick reply and helpful! Thanks Justin!
